### PR TITLE
For #175: Adding Plugins API interface.

### PR DESCRIPTION
**Pull request by bkuzmic**
For #175:

- adding Plugins API initial interface

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#217](https://github.com/amihaiemil/docker-java-api/issues/217) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#217](https://github.com/amihaiemil/docker-java-api/issues/217) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/f3151f14-a85c-487a-b54d-fcfcea3574a0); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/amihaiemil/docker/RtDocker.java [hunk 1]
```diff
@@ -129,6 +129,15 @@ public DockerSystem system() {
         );
     }
 
+
+    @Override
+    public Plugins plugins() {
+        throw new UnsupportedOperationException(
+            "Plugins API is not yet implemented. If you can contribute please,"
```
Comment (paulodamaso) [inline src/main/java/com/amihaiemil/docker/RtDocker.java]: @bkuzmic Let's avoid to make string concatenations with the `+` operator; here you can use `String.join()`. Besides, this comma after "please" could be removed.

File: src/main/java/com/amihaiemil/docker/Plugins.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+/**
+ * Plugins API.
+ * @author Boris Kuzmic (boris.kuzmic@gmail.com)
+ * @since 0.0.7
+ * @todo #208:30min Continue implementing Plugins API methods. More information
+ *  about available methods:
+ *  https://docs.docker.com/engine/api/v1.35/#tag/Plugin
+ */
+public interface Plugins {
+
```
Comment (paulodamaso) [inline src/main/java/com/amihaiemil/docker/Plugins.java]: @bkuzmic You could at least declare `Plugin` methods based on specs.

**Comment by paulodamaso** [comment]
@bkuzmic Great, thanks.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Job was finished in 24 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#217](https://github.com/amihaiemil/docker-java-api/issues/217) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**